protos/backend_service.proto

```diff
@@ -168,4 +171,47 @@ message PingRequest {
 // Response payload for ping operations.
 message PingResponse {
 	// No fields
-}
\ No newline at end of file
+}
+
+// Request payload for fetching backend metrics.
+message GetMetricsRequest {
+    // No fields
+}
+
+// Response payload for fetching backend metrics
+message GetMetricsResponse {
+    // The current metrics for the backend service.
+    BackendMetrics metrics = 1;
+}
+
+// Metrics for the backend service.
+message BackendMetrics {
+    // Activity work item metrics
+    WorkItemMetrics activityWorkItems = 1;
```

Comment:
Since these messages are often encoded as JSON too, i'd recommend adding explicit `json_name` tags so we contractualize the expected JSON format in here too, and avoid ambiguities (also because the default for protobuf is snake_case, while JS developers prefer camelCase): https://stackoverflow.com/a/61818138/192024